addons/grenades/CfgAmmo.hpp

```diff
@@ -124,4 +124,26 @@ class CfgAmmo {
         soundImpactWoodExt3[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.1,75};
         soundImpactWoodExt4[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.15,75};
     };
+
+    class ACE_G_Incendiary: SmokeShell {
```

Comment:
`ACE_G_Flashbang2` is a bad name.
Name would depend on the situation. If it's a Soviet stun grenade then maybe: `ACE_G_Flashbang_Opfor`. If it's a `DM109` then mybe `ACE_G_Flashbang_Germany`